frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -360,6 +360,24 @@ export default {
       })
       return applicationItems
     },
+
+    //TODO - Add Supp App PDF function will look very similar, but it will hit a seperate endpoint
+    //the supp app PDF's do not generate yet in Dynamics, so holding off on including that code
+    async downloadPDF(application) {
+      if (application.applicationType === 'OFM') {
```

Comment:
we may want to add a try/catch here and show failure alert for download pdf failed

Reply:
good call - fixed